Dockerfile

```diff
@@ -1,6 +1,11 @@
-FROM ${AIRFLOW_IMAGE_NAME:-apache/airflow:2.1.2}
-COPY poetry.lock poetry.lock
-COPY pyproject.toml pyproject.toml 
+FROM ${AIRFLOW_IMAGE_NAME:-apache/airflow:2.2.0-python3.7}
+
+USER airflow
+
+ENV PYTHONPATH "${PYTHONPATH}:/data_pipelines/"
+
+COPY --chown=airflow:airflow poetry.lock pyproject.toml ./
```

Comment:
This is a more efficient way to copy the files into our container. 
It also changes the owner to the airflow user